### DISCUSSION TITLE
History Title Does Not Match Logfile

### DISCUSSION THREAD

**Discussion by trleith** [automatic-ripping-machine/automatic-ripping-machine#865] [General]
I don't know how else to describe this. Is there something I can do to fix it?
See attached picture
![LogfileHistoryMismatch](https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/59936316/c71032a9-c145-4b40-9c89-37641c4fee91)

**Comment by microtechno9000** [comment]
Are they music files/rips?

if so, I have noticed recently with music that something strange is happening when labelling music files and the logs. Havent had time to investigate yet

**Reply by trleith** [reply]
Yes, music. DVD & BluRay seem to be fine

**Comment by microtechno9000** [comment]
I have raised #866 to capture the issue, welcome to add any additional information to help resolve.

**Reply by trleith** [reply]
OK, thanks.
